backend/src/main/java/gov/cdc/usds/simplereport/api/model/errors/InvalidPatientLinkException.java

```diff
@@ -1,35 +1,12 @@
 package gov.cdc.usds.simplereport.api.model.errors;
 
-import graphql.ErrorClassification;
-import graphql.GraphQLError;
-import graphql.language.SourceLocation;
-import java.util.Collections;
-import java.util.List;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.ResponseStatus;
 
 @ResponseStatus(
-    code = HttpStatus.UNAUTHORIZED,
+    code = HttpStatus.FORBIDDEN,
```

Comment:
The `PreAuthorize` that is _actually_ responsible for the status codes that end up representing this state actually throws a 403, so I'm just updating this to match

Reply:
So do we get the reason from the annotation but a 403 code, right now?

Reply:
Is that just a hard and fast rule of `@PreAuthorize` annotations (that they always resolve failures to 403 responses)?

Reply:
We do not get the reason, @benwarfield-usds. And yes @jeskew-gov, that's just how `@PreAuthorize` annotations work as far as I understand the documentation